docs/tutorials/common_data_operations.ipynb

```diff
@@ -141,7 +141,9 @@
    "source": [
```

Comment:
<p>Some kind of callout box for this would be cool</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Ah that looks much better thank you